### ISSUE TITLE
Agent does not log stderr from the Node-RED process

### ISSUE THREAD

**Issue by knolleary** [flowforge/flowforge-device-agent#21]
### Current Behavior

The launcher does not currently log anything from stderr. This makes it very hard to debug issues where the process fails to start (see #20 for an example).

### Expected Behavior

We should be capturing and logging both stderr and stdout.

### Steps To Reproduce

Until we fix #20 , run the agent with `--dir` set to a relative path with the device registered and assigned to a project with an active snapshot so it tries to start node-red.

### Environment

- FlowForge version:
- Node.js version:
- npm version:
- Platform/OS:
- Browser:

**Comment by hardillb** [comment]
There looks to be code to gather stderr the same was as stdout in the launcher: 

https://github.com/flowforge/flowforge-device-agent/blob/dae1227517f247f94896c66deb0f6cfc53228671/lib/launcher.js#L200-L214

Both end up in the same buffer

**Comment by hardillb** [comment]
@knolleary This looks like it should already be fixed.

I edited a `console.error('Ben Testing')` into `node-red/red.js` and I get the expected output.

```
node index.js -d foo
[AGENT] 2 Sept 2022, 18:17:52 info: FlowForge Device Agent
[AGENT] 2 Sept 2022, 18:17:52 info: Version: 0.2.2
[AGENT] 2 Sept 2022, 18:17:52 info: Device: 0WR30bz6nX
[AGENT] 2 Sept 2022, 18:17:52 info: ForgeURL: http://localhost:3000
[AGENT] 2 Sept 2022, 18:17:52 info: Project: 15bf9f7c-10a0-4abb-b64b-ed384c582054
[AGENT] 2 Sept 2022, 18:17:52 info: Snapshot: 47E0MWzNxZ
[AGENT] 2 Sept 2022, 18:17:52 info: Settings: none
[AGENT] 2 Sept 2022, 18:17:52 info: Starting http poll thread. Interval: 30s
[AGENT] 2 Sept 2022, 18:17:52 info: Starting project
[NR] Ben Testing
[NR] 2 Sep 18:17:52 - [info] 
[NR] Welcome to Node-RED
[NR] ===================
[NR] 2 Sep 18:17:52 - [info] Node-RED version: v3.0.2
[NR] 2 Sep 18:17:52 - [info] Node.js  version: v16.15.0
[NR] 2 Sep 18:17:52 - [info] Linux 5.19.4-100.fc35.x86_64 x64 LE
[NR] 2 Sep 18:17:52 - [info] Loading palette nodes
[NR] 2 Sep 18:17:53 - [warn] Project Link nodes require flowforge settings
[NR] 2 Sep 18:17:53 - [info] Settings file  : /opt/share/projects/flowforge/flowforge-dev-env/packages/flowforge-device-agent/foo/project/settings.js
[NR] 2 Sep 18:17:53 - [info] Context store  : 'default' [module=memory]
[NR] 2 Sep 18:17:53 - [info] User directory : /opt/share/projects/flowforge/flowforge-dev-env/packages/flowforge-device-agent/foo/project
[NR] 2 Sep 18:17:53 - [warn] Projects disabled : set editorTheme.projects.enabled=true to enable
[NR] 2 Sep 18:17:53 - [info] Flows file     : /opt/share/projects/flowforge/flowforge-dev-env/packages/flowforge-device-agent/foo/project/flows.json
[NR] 2 Sep 18:17:53 - [info] Admin UI disabled
[NR] 2 Sep 18:17:53 - [info] Server now running at http://127.0.0.1:1880
[NR] 2 Sep 18:17:53 - [warn] Encrypted credentials not found
[NR] 2 Sep 18:17:53 - [info] Starting flows
[NR] 2 Sep 18:17:53 - [info] Started flows
```

**Comment by knolleary** [comment]
It would appear I went on to fix this as part of #22 

https://github.com/flowforge/flowforge-device-agent/commit/905cd5681023c4a71715b9972453b0103909d2d6#diff-a4ef8d754c46f33ab02a620e91925ac615d7412f2f80edf06b2425cba5ff3104R192-R193
